### ISSUE TITLE
logits to percent for probability raster

### ISSUE THREAD

**Issue by FredericSorbe** [LUP-LuftbildUmweltPlanung/SITS_classification#14]
Currently probability raster is stored with logit values in float322 datatype format. Logits should be transformed to percentage values and stored as int16.
